pkg/scheduler/metrics/metrics.go

```diff
@@ -235,3 +244,7 @@ func UnschedulablePods() metrics.GaugeMetric {
 func SinceInSeconds(start time.Time) float64 {
 	return time.Since(start).Seconds()
 }
+
+func UnschedulableReason(plugin string, profile string) metrics.GaugeMetric {
+	return unschedulableReasons.With(metrics.Labels{"plugin": plugin, "profile": profile})
```

Comment:
nit: using `WithLabelValues` is slightly better in terms of performance since it doesn't require creating a new object every time.

Reply:
@dgrisonnet This seems to be giving an error to me locally, give me some time to test this